lib/contestant.rb

```diff
@@ -0,0 +1,32 @@
+require "./lib/game"
+require 'pry'
+
+class Contestant
+attr_reader :first_name, :last_name, :age, :state_of_residence, :spending_money, :game_interests_list
```

Comment:
No need for an attr_reader for first_name or last_name here, because we only ever need to call .full_name.